WarOfEmpires/Views/Empire/Troops.cshtml

```diff
@@ -46,6 +46,24 @@
     @Html.HiddenFor(m => m.CavalryTrainingCost)
     @Html.HiddenFor(m => m.FootmanTrainingCost)
     @Html.HiddenFor(m => m.MercenaryTrainingCost)
+    @Html.HiddenFor(m => m.CurrentStamina)
+
+    <div class="row">
+        <div class="col-4 col-form-label">
+            @Html.LabelFor(m => m.Stamina)
+        </div>
+        <div class="col-2 col-form-label">
+            @Model.CurrentStamina.ToString(StringFormat.Integer)
+        </div>
+        <div class="col-2">
+            @Html.TextBoxFor(m => m.StaminaToFull, new { @class = "form-control" })
```

Comment:
Do you eventually really need both StaminaToFull and uh, the other one? MaxCanAfford? So they will probably be merged somehow later.

Reply:
Yes, they will be merged somehow, to use the largest one